cpu/sam21_common/Makefile.include

```diff
@@ -1,6 +1,8 @@
-# export the CPU family so we can differentiate between them in the code
-FAM = $(shell echo $(CPU_FAM) | tr 'a-z-' 'A-Z_')
-export CFLAGS += -DCPU_FAM_$(FAM)
+# Define the CPU family so we can differentiate between them in the code
+CFLAGS += -DCPU_FAM_$(shell echo $(CPU_FAM) | tr 'a-z-' 'A-Z_')
+
+# Define the SAM0 family so we can differentiate between them in the code
+CFLAGS += -DSAM0_FAM_$(shell echo $(SAM0_FAM) | tr 'a-z-' 'A-Z_')
```

Comment:
I looked at it, but the problem is that `CPU_FAM` is currently set in `cpu/samd21/Makefile.include`. And that one is shared for samd21 and samr21 boards. We need a parameter to distinguish between these two.

Notice that for samr21-xpro `SAM0_FAM` is set to `samr21`.

That why I came up with `SAM0_FAM` which must be set in the board configuration.